x-pack/plugins/triggers_actions_ui/public/common/expression_items/of.scss

```
@@ -0,0 +1,3 @@
+.actOf__aggFieldContainer {
+  width: 480px;
```

Thanks for putting this on its own scss file. Remember that we always avoid directly using a pixel value. Use something like instead
```suggestion
  width: $euiSize * 30;
```